assets/prototype/domain/eventEditor/data/mutations/datetimes/test/updateDatetime.test.ts

```diff
@@ -121,7 +121,7 @@ describe('updateDatetime', () => {
 			}
 		);
 
-		const tempTicketId: string = 'temp-tkt-id';
+		const tempTicketId = 'temp-tkt-id';
```

Comment:
Captain Obvious has been killed here.

Reply:
:smile:

Reply:
I am curious if and how TS does type inference based on constant content.

Reply:
Because every primitive type is an object which contains the information like `typeof` etc.